templates/third-party-post.adoc

```diff
@@ -18,6 +19,8 @@ AUTHOR_NAME <https://github.com/GITHUB_USERNAME>
 // Above:
 // Do not insert any blank lines between any of the lines above.
 //
+// "open-graph-image" is set to OL logo, howver it can be changed to a different image or left blank in which case it will be set to the default  // Open Liberty logo
```

Comment:
formatting of comment seems incorrect
add comment that whenever possible, a more appropriate/specific image should be used. For example, if present, to an image used within the blog post.

Reply:
Done